components/admin/requests/modals/CancelCreateRequestModal.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  Button,
+  Text,
+  Box,
+  useDisclosure,
+} from '@chakra-ui/react'; // Chakra UI
+import Link from 'next/link'; // Link
+import { ReactNode } from 'react'; // React JSX Type
+
+type CancelRequestModalProps = {
+  readonly type: 'replacement' | 'renewal';
+  readonly children: ReactNode;
+};
+
+export default function CancelCreateRequestModal({ type, children }: CancelRequestModalProps) {
```

Comment:
Could you add a small jsdoc comment for this component, would go a long way for reusability!